BuildInformation.md

```diff
@@ -0,0 +1,7 @@
+# Quality Tools for BlueJ (QT4BJ) - Build Information
+
+## Required Java version(s)
+Your JDK version should match the requirements of your version of BlueJ. See the [main BlueJ website](https://www.bluej.org/) or [BlueJ installations](https://www.bluej.org/versions.html) to verify that your JDK version is correct
```

Comment:
Please use third-person rather than first-person 
```suggestion
The JDK version should match the requirements of the targeted version of BlueJ. See the [main BlueJ website](https://www.bluej.org/) or [BlueJ installations](https://www.bluej.org/versions.html) to verify the correct JDK version to use.
```

Where is the necessary "JDK version" information at either of the given URLs?  These references need to be specific enough that maintainers get the information claimed as available.

Reply:
I have changed this statement to 3rd person and added information about the necessary JDKs being packaged with the BlueJ installations in my most recent commit.